packages/upswyng-web/src/components/AlertPanel.tsx

```diff
@@ -0,0 +1,78 @@
+import ExpansionPanel, {
+  ExpansionPanelProps,
+} from "@material-ui/core/ExpansionPanel";
+import Avatar from "@material-ui/core/Avatar";
+import ExpandMore from "@material-ui/icons/ExpandMore";
+import ExpansionPanelDetails from "@material-ui/core/ExpansionPanelDetails";
+import ExpansionPanelSummary from "@material-ui/core/ExpansionPanelSummary";
+import Grid from "@material-ui/core/Grid";
+import MuiMarkdown from "./MuiMarkdown";
+import React from "react";
+import { TAlertFull } from "@upswyng/upswyng-types";
+import { Theme } from "@material-ui/core/styles/createMuiTheme";
+import Tooltip from "@material-ui/core/Tooltip";
+import Typography from "@material-ui/core/Typography";
+import makeStyles from "@material-ui/styles/makeStyles";
+
+// const AlertPanelTooltip = ({children}:{})
```

Comment:
missed that guy

Reply:
oh derp. [upderped](https://github.com/CodeForBoulder/upswyng/pull/309/commits/cd1925966e22eb8b36c1c0c9c302d99d295e8cc3)